packs/pathfinder-monster-core/bogwid.json

```
@@ -171,7 +171,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The larvae launched from the bogwid attach themselves to the target and begin to feed. Once a larva is attached, the target becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1}. While the larva remains attached, the target cannot recover from persistent bleed. To remove the larva, the target can attempt a DC 21 @UUID[Compendium.pf2e.actionspf2e.Item.Escape] check. Additionally, any area damage dealt to the target destroys all attached larvae.</p>\n<hr />\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Ravenous Young]</p>"
+                    "value": "<p>The larvae launched from the bogwid attach themselves to the target and begin to feed. Once a larva is attached, the target becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1}. While the larva remains attached, the target cannot recover from persistent bleed. To remove the larva, the target can attempt [[/act escape dc=21]]. Additionally, any area damage dealt to the target destroys all attached larvae.</p>\n<hr />\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Ravenous Young]</p>"
```

```suggestion
                    "value": "<p>The larvae launched from the bogwid attach themselves to the target and begin to feed. Once a larva is attached, the target becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1}. While the larva remains attached, the target cannot recover from persistent bleed. To remove the larva, the target can attempt to [[/act escape dc=21]]. Additionally, any area damage dealt to the target destroys all attached larvae.</p>\n<hr />\n<p>@UUID[Compendium.pf2e.bestiary-effects.Item.Effect: Ravenous Young]</p>"
```

"attempt to escape"/"attempt an escape check" I'm ok with either